pallets/loans/src/functions.rs

```diff
@@ -70,20 +71,16 @@ impl<T: Config> Pallet<T> {
 		);
 
 		// create new loan nft
-		let loan_pallet_account: T::AccountId = T::LoansPalletId::get().into_account();
```

Comment:
Is the `LoansPalletId` used anywhere else to hold something or can we remove it?

Reply:
It's used here: https://github.com/centrifuge/centrifuge-chain/blob/loans-cleanup/pallets/loans/src/functions.rs#L38 Currently the loan NFTs are transferred here, and it's also used by the tests to find the pallet account. We can probably consider removing this, but I'd rather do that in a separate PR.